extensions/wikia/CommunityPage/scripts/entryPoint.js

```diff
@@ -0,0 +1,24 @@
+require([
+	'jquery',
+	'wikia.tracker',
+], function ($, tracker) {
+	'use strict';
+
+	var track = tracker.buildTrackingFunction({
+		action: tracker.ACTIONS.CLICK,
+		category: 'community-page-entry-point',
+		trackingMethod: 'analytics'
+	});
+
+	$('.community-page-entry-point-module').on('mousedown touchstart', 'a', function () {
```

Comment:
I think we can bind event to '.community-page-entry-point-module .entry-point-button' directly